charts/rancher-turtles/values.yaml

```diff
@@ -3,5 +3,14 @@ rancherTurtles:
   tag: v0.0.0
   imagePullPolicy: Never
   namespace: rancher-turtles-system
+capi:
+  enabled: true
+  namespace: capi-system
+  configSecret:
```

Comment:
Should this be included before 0.5.2 version is released?

Reply:
v1alpha2 is for v0.6.0 we can reuse the same values, the difference will be in the provider manifest, we also have conversion webhooks to handle the transition.